code/controllers/subsystem/parallax.dm

```diff
@@ -4,11 +4,21 @@ SUBSYSTEM_DEF(parallax)
 	flags = SS_POST_FIRE_TIMING | SS_BACKGROUND
 	priority = FIRE_PRIORITY_PARALLAX
 	runlevels = RUNLEVEL_LOBBY | RUNLEVELS_DEFAULT
-	var/list/currentrun
+	var/current_run_pointer = 1
+	var/list/currentrun = list()
+	var/list/queued = list()
 	var/planet_x_offset = 128
 	var/planet_y_offset = 128
 	var/random_layer
 	var/random_parallax_color
+	//Amount of ticks between the parallax being allowed to freely fire without going into the queue
+	var/parallax_free_fire_delay_ticks = 10
```

Comment:
Is this modified? If not, why isn't this a define?
Also don't we already have some sort of fire var, where we set how often does it run?

Reply:
Theres no point making this a define, its only ever used once and the value is clearly defined (The numeber of tickets between the parallax being able to fire freely)

Reply:
The var is never modified, thus it has no reason to be a var and should be a define